src/components/Tabs/Tabs.tsx

```diff
@@ -61,6 +61,7 @@ function TabsComponent({
         format={format}
         index={i}
         label={props.label}
+        dataId={props.dataId}
```

Comment:
The` <li>` element surrounding the NavItem is actually what handles the click event. It may be best to put the data-id on that element because that is what the user will interact with for tracking, and it can also be used to trigger clicks in testing.